app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,24 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self._skill_level
+
+    def player_info(self) -> str:
+        return (
+            f"Dwarf blacksmith {self.nickname} "
+            f"with skill of the "
+            f"{self._skill_level} "
+            f"level"
+        )
+
+    def eat_favourite_dish(self) -> str:
```

Comment:
The `eat_favourite_dish` method is defined to return a `str`, but it currently does not return any value. Consider returning a string or changing the return type to `None`.